python_modules/dagster/dagster/_daemon/controller.py

```diff
@@ -271,11 +285,23 @@ def check_daemon_heartbeats(self) -> None:
             )
 
     def check_workspace_freshness(self, last_workspace_update_time: float) -> float:
+        if self._skip_workspace_reload:
```

Comment:
I think we can remove this and just add the env var to prevent the registry from reloading - i don't see a downside to the daemon doing the refresh periodically without reloading, and it keeps things simpler to only have one configuration option